### PR TITLE
Card display is done for the player and table

### PR DESCRIPTION
**Pull request by dhillonJas**
Cards are visible on the table and in player's hand

### CODE REVIEW

File: digital-deck-ui/src/store/index.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ export default createStore({
     state: {
         sessionId: -1,
         playerId: -1,
-        playerCards: null,
-        tableCards: null,
+        playerCards: [],
```
Comment (mdangnguyen) [inline digital-deck-ui/src/store/index.js:11]: I set these to null so the tests can check whether `initSession` actually grabbed the data. Do you need these to be empty arrays for any reason?

File: digital-deck-ui/src/store/index.js [hunk 2]
```diff
@@ -31,22 +31,29 @@ export default createStore({
...
+                cardsPerPlayer: 3,
+                cardsOnTable: 10
             }).then((res) => {
                 commit('setSessionId', res.data.sessionId);
                 commit('setPlayerId', res.data.players[0].playerId);
                 commit('setPlayerCards', res.data.players[0].cards);
                 commit('setTableCards', res.data.table);
                 commit('setCardsInDeck', res.data.deck);
 
+                console.log(state.playerCards);
```
Comment (mdangnguyen) [inline digital-deck-ui/src/store/index.js:56]: I don't think you wanted to commit this

File: digital-deck-ui/src/components/CardDeckImageEnum.js [hunk 3]
```diff
@@ -1,9 +1,66 @@
...
-    FULL: 'https://i.imgur.com/Ggu8xlE.png',
-    HALF: 'https://i.imgur.com/4pqATmt.png',
-    FIVE: 'https://i.imgur.com/fP9bL6O.png',
-    FOUR: 'https://i.imgur.com/tPJcCL9.png',
-    THREE: 'https://i.imgur.com/83dtG4X.png',
-    TWO: 'https://i.imgur.com/zIQRKqa.png',
-    ONE: 'https://i.imgur.com/TZJdpPC.png'
-};
\ No newline at end of file
+    BACK: {
```
Comment (mdangnguyen) [inline digital-deck-ui/src/components/CardDeckImageEnum.js:2]: I originally intended for this file to just be assets for the card deck. I think you should either put the assets for the face of the cards in a different file or rename this file to something like `CardImageEnum.js` to be more accurate.

File: digital-deck-ui/src/store/index.js (hunk 1 above)
Comment (dhillonJas) [reply digital-deck-ui/src/store/index.js:11]: The previous method I was working with showed some weird behavior with them being null. I changed my approach later but forgot to change these later.

File: digital-deck-ui/src/components/CardDeckImageEnum.js (hunk 3 above)
Comment (dhillonJas) [reply digital-deck-ui/src/components/CardDeckImageEnum.js:2]: I think the name still implies to whole deck, be it front or back

File: digital-deck-ui/src/components/PlayerHand.vue [hunk 4]
```diff
@@ -1,12 +1,37 @@
 <template>
-    <div>
-        <div>Display player's cards here</div>
-        <div>Current hand (For testing): {{ $store.getters.getPlayerCards }}</div>
+  <div id="player-area">
```
Comment (mdangnguyen) [inline digital-deck-ui/src/components/PlayerHand.vue:2]: Are you using this id anywhere?

Comment (dhillonJas) [reply digital-deck-ui/src/components/PlayerHand.vue:2]: I put this so it was easier to tell player-area when dubbing with inspect. No need for it now

File: digital-deck-ui/src/components/CardDeckImageEnum.js (hunk 3 above)
Comment (mdangnguyen) [reply digital-deck-ui/src/components/CardDeckImageEnum.js:2]: I guess it's fine. The name just doesn't really click with me

**Pull request merged**